### PR TITLE
search bar

### PR DESCRIPTION
**Pull request by tatiana-sadaeva**

### CODE REVIEW

**Comment by tatiana-sadaeva** [comment]
[DEMO LINK](https://tatiana-sadaeva.github.io/layout_search-bar-airbnb/)
[TEST REPORT LINK](https://tatiana-sadaeva.github.io/layout_search-bar-airbnb/report/html_report/)

File: src/index.html [hunk 1]
```diff
@@ -11,6 +11,52 @@
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>Search bar airbnb</h1>
+    <h1></h1>
```
Comment (DarkMistyRoom) [inline src/index.html:14]:
```suggestion
```

File: src/index.html [hunk 2]
```diff
@@ -11,6 +11,52 @@
...
+            type="text"
+            name="bigSearchBar"
+            id="big"
+            placeholder="Try &quot;Los Angeles&quot;"
+          >
+          <img
+          src="./images/Search.svg"
+          alt="search"
+          class="bigButton"
+        >
```
Comment (DarkMistyRoom) [inline src/index.html:35]:
```suggestion
          <img
            src="./images/Search.svg"
            alt="search"
            class="bigButton"
          >
```

File: src/index.html [hunk 3]
```diff
@@ -11,6 +11,52 @@
...
+            src="./images/Search.svg"
+            alt="search"
+            class="smallButton"
+          >
+        </div>
+      </form>
+    </div>
   </body>
-</html>
+</html>
```
Comment (DarkMistyRoom) [inline src/index.html:62]: add the empty line at the very end

File: src/style.css [hunk 4]
```diff
@@ -0,0 +1,117 @@
...
+  font-family: "Avenir";
+  src: url(./fonts/Avenir-Book.ttf);
+  src: url(./fonts/Avenir-Heavy.ttf);
+  src: url(./fonts/Avenir.ttc);
+}
+
+/* body {
+  padding: 0;
+  margin: 0;
+} */
```
Comment (DarkMistyRoom) [inline src/style.css:11]: remove the comments

File: src/style.css [hunk 5]
```diff
@@ -0,0 +1,117 @@
...
+
+.page {
+  box-sizing: border-box;
```
Comment (DarkMistyRoom) [inline src/style.css:14]: move `  box-sizing: border-box;` to the `* `selector

File: src/style.css [hunk 6]
```diff
@@ -0,0 +1,117 @@
...
+  line-height: 22px;
+  font-size: 16px;
+  border: 1px solid #e1e7ed;
+  box-shadow: 0 1px 8px rgba(61, 78, 97, 0.1);
+  border-radius: 4px;
+  padding: 23px 0 23px 62px;
+}
+
+.bigBar::placeholder {
+  font-family: Avenir, sans-serif;
```
Comment (DarkMistyRoom) [inline src/style.css:46]: move `font-family` to the `html `selector and inherit where needed

File: src/style.css [hunk 7]
```diff
@@ -0,0 +1,117 @@
...
+  height: 100%;
+  border: 1px solid #e1e7ed;
+  box-shadow: 0 1px 8px rgba(61, 78, 97, 0.1);
+  border-radius: 4px;
+  padding: 10px 0 10px 33px;
+  font-family: Avenir, sans-serif;
+  font-size: 14px;
+  font-weight: 300;
+  line-height: 19px;
+}
```
Comment (DarkMistyRoom) [inline src/style.css:82]: check the 2nd topic in the checklist. Don't repeat yourself

**Review by DarkMistyRoom — CHANGES_REQUESTED** [review]
Remove this outline
image

**Pull request closed without merging**